MainModule/Server/Commands/Donors.lua

```diff
@@ -37,6 +37,59 @@ return function(Vargs, env)
 			end
 		};
 
+		DonorTShirt = {
+			Prefix = Settings.PlayerPrefix;
+			Commands = {"tshirt", "givetshirt"};
```

Comment:
```suggestion
			Commands = {"tshirt", "givetshirt", "donortshirt"};
```
should have a donortshirt command (some games set `Settings.Prefix` and `Settings.PlayerPrefix` to the same character so we should add something in for these cases)